contracts/svm/programs/express_relay/src/utils.rs

```diff
@@ -49,6 +57,29 @@ pub fn transfer_lamports_cpi<'info>(
     Ok(())
 }
 
+pub fn transfer_spl<'info>(
+    from_ta: &AccountInfo<'info>,
+    to_ta: &AccountInfo<'info>,
+    token_program: &AccountInfo<'info>,
+    authority: &AccountInfo<'info>,
+    mint: &InterfaceAccount<'info, Mint>,
+    amount: u64,
+) -> Result<()> {
+    let cpi_accounts = SplTransfer {
+        from:      from_ta.clone(),
+        to:        to_ta.clone(),
+        mint:      mint.to_account_info(),
+        authority: authority.clone(),
+    };
+
+    token_interface::transfer_checked(
```

Comment:
nit, we can save some gas here by checking `amount > 0` first

Reply:
done